.circleci/main.yml

```diff
@@ -1163,7 +1163,61 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/mount-server/mount-server_${CIRCLE_TAG:1}* --generate-notes
             fi
+  console-release-draft:
+    docker:
+      - image: cimg/base:stable
+    steps:
+      - gh/setup:
+          version: 2.13.0
+      - run:
+          name: create github console release draft
+          command: |
+            if [[ $CIRCLE_TAG == *"-"* ]];
+              then
+                  echo "git tag has - assuming its a prerelease."
+                  parentBranch=master
+              else
+                  echo "regular release."
+                  parentBranch=${CIRCLE_TAG:1}
+              fi
+            git clone https://github.com/pachyderm/haberdashery.git haberdashery
+            git checkout -b "release-${CIRCLE_TAG:1}"
+            tmp=$(mktemp)
+            jq --arg val "${CIRCLE_TAG:1}-${CIRCLE_SHA1}" '.pachyderm = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_SHA1}" '.pachReleaseCommit = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_TAG:1}" '.pachctl = $val' version.json > "$tmp" && mv "$tmp" version.json
+            jq --arg val "${CIRCLE_SHA1}" '.proto = $val' version.json > "$tmp" && mv "$tmp" version.json
```

Comment:
you might not want to bump the proto. I think this field should be left alone and only updated by developers when we need new fields or functions from core.

Reply:
you are also bumping without building the protos in the script.

Reply:
good catch!